### DISCUSSION TITLE
Cannot find my website in the google search

### DISCUSSION THREAD

**Discussion by georgemilosh** [alshedivat/al-folio#1265] [Q&A] [answered]
The website I made based on this repo is accessible, but when I search for it in Google I cannot find it. It has been up for quite some time (months). Is there some reason?

**Comment by george-gca** [comment, marked as answer]
Answered in #1514

**Reply by pdeperth** [reply]
Hello!
Sorry, I did not get the answer: this links to a question about centering a figure right, not about Google searches?
